spinta/datasets/commands/check.py

```diff
@@ -1,14 +1,45 @@
 from spinta import commands
 from spinta.components import Context
 from spinta.datasets.components import Dataset, Resource
+import re
 
 
 @commands.check.register()
 def check(context: Context, dataset: Dataset):
     for resource in dataset.resources.values():
         commands.check(context, resource)
 
+    rc = context.get('rc')
+
+    if rc.get('names'):
+        name = dataset.name
+        is_lowercase = len([i for i in name if i.islower()] + re.findall('/', name)) == len(name)
```

Comment:
Can you check name with only regexps? A one regex, like this:

    [a-z][a-z0-1_]*(/[a-z][a-z0-1_]*)*

Should check whole dataset name.

Reply:
https://regexkit.com/python-regex